feat(content) Rulei Contact Conversation (#7995)
endless-sky/endless-sky: data/rulei/rulei.txt

	invisible
	landing
	to complete
		never
	npc save
		government "Rulei"
		personality heroic vindictive staying mute
		system "Kanguwa"
		ship "vyu-Ira" "Solemnity"

mission "Ira Contact"
	invisible
	landing
	to complete
		never
	on enter Kanguwa
		fail
		log "Factions" "Rulei" `The Rulei were likely an ancient technologically advanced civilization from long ago.`
			`They communicate through some sort of invasive psychonic ability.`
		conversation
			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. The best way to describe it is an overwhelming sense of emptiness, but at the same time, the intensity of it makes it feels like something must be there. You barely have enough time to try and make sense of this oddity as only seconds later, you start to hear ominous whispering, as if something is behind you.`
			choice
				`	(Turn around.)`
			`	You turn your head around to look, but every time, the source of the whispers remains at the back of your head, like an itch that can't be reached. You can't even remotely understand the language, and the whispers progressively increase in volume. The louder they get, the more there are.`
			

> 			`It sounds like thousands of varingly pitched voices are speaking at all at once now, and you senses all feel overloaded. Just before you think you can't take anymore, everything falls completely silent.`

Reviewer: ```suggestion
			`It sounds like thousands of varingly pitched voices are speaking at all at once now, and you senses all feel overloaded. Just before you think you can't take anymore, everything falls completely silent.`
			choice
				`    (Wait for something to happen.)`
```

Not too sure but something like that?

> 			scene "scene/rulei flash"

Reviewer: From what I experienced it is best to have a choice node before a scene, it just looks better

Author: If you can think of something to put there, sure, but I can't think of anything that would fit without being weird.

# Copyright (c) 2022 by Saugia
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Relic of the Past"
	invisible
	landing
	to complete
		never
	npc save
		government "Rulei"
		personality heroic vindictive staying mute unconstrained frugal
		system "Kanguwa"
		ship "vyu-Ira" "Solemnity"

mission "Ira Contact"
	invisible
	landing
	to complete
		never
	on enter Kanguwa
		fail

> 		log "Factions" "Rulei" `The Rulei were likely an ancient technologically advanced civilization from long ago.`

Reviewer 2: ```suggestion
		log "Factions" "Rulei" `The Rulei were likely an ancient, technologically advanced civilization from long ago.`
```

> 			`They communicate through some sort of invasive psychonic ability.`

Reviewer 3: ```suggestion
			`They communicate through some sort of invasive psionic ability.`
```

		conversation

> 			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You barely have enough time to try and make sense of this oddity as only seconds later you start to hear ominous whispering come from behind you.`

Author: I'll see what I can add on for it.

> 			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You barely have enough time to try and make sense of this oddity as only seconds later, you start to hear ominous whispering, as if something is behind you.`

Reviewer 2: ```suggestion
			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You barely have enough time to try and make sense of this oddity as only seconds later you start to hear ominous whispering, as if something is behind you.`
```

> 			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You barely have enough time to try and make sense of this oddity as only seconds later you start to hear ominous whispering, as if something is behind you.`

Reviewer 3: ```suggestion
			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You have barely any time to try and make sense of this oddity, though; only seconds later you start to hear ominous whispering, as if something is behind you.`
```
That last sentence and the double "as" is feeling pretty awkward. Attempted to resolve with smallest possible change, but a more extensive one may be better.

Author: Yeah, I think maybe I should try and expand this a bit more rather than try and keep it shorter and change it only for the sake of the double "as". I'll think on it.

			choice

> 				`	(Turn around.)`

Author: I'm assuming in adding other options, you mean to have different things unfold to, as if I remember correctly you don't like several choices lead to one answer. The choices are to break up the dialogue so that it's not just presented as one large text dump which is off-putting in many cases, and it's more narratively presentable if it's done more in segments. I can try to add different options, but it might be hard considering there isn't a whole lot of different things that can happen when something is whispering in your head.

> 				`	(Turn around.)`

Reviewer 4: Having two options that lead to the same text is better than only having a single option.

> 				`	(Turn around.)`

Author: ALright. I wasn't sure if that was also a no-no, but if it could fit here, then I would prefer to do that to split up dialogue than not.

> 			`	You turn your head around to look, but the source of the whispers remains at the back of your head, like an itch that can't be reached. You can't even remotely understand the language, and the whispers progressively increase in volume. The louder they get, the more there are. They begin to come from various directions around you, as though they're tugging at your mind from several angles at once. It sounds like hundreds of differently pitched voices are speaking all at once now, and your senses all feel overloaded. Just before you think you can't take anymore, everything falls completely silent.`

Author: I think this is too many "whispers" in a row, which is what I was trying to avoid previously (three sentences in a row). I'll try to think of a way to change it, but the rest is good.

			choice
				`	(Wait.)`
			scene "scene/rulei flash"
			`	"T e r r a n."`

> 			`A bizarre image is suddenly flashed into your head, and it feels like millions of eyes are starting at you beyond the boundaries of space. Turning your head to look around again, you find the source - a strange-looking spacecraft in the vicinity of two of this system's planets. When it spoke, it sounded like thousands of voices speaking at once again, but only for a brief second this time. Still, the various whispers surround you everywhere else.`

Reviewer 5: ```suggestion
			`A bizarre image is suddenly flashed into your head, and it feels like millions of eyes are staring at you beyond the boundaries of space. Turning your head to look around again, you find the source - a strange-looking spacecraft in the vicinity of two of this system's planets. When it spoke, it sounded like thousands of voices speaking at once again, but only for a brief second this time. Still, the various whispers surround you everywhere else.`
```

> 			`A bizarre image is suddenly flashed into your head, and it feels like millions of eyes are staring at you from beyond the boundaries of space. Turning your head to look around again, you find the source - a strange-looking spacecraft in the vicinity of two of this system's planets. When it spoke, it sounded like thousands of voices speaking at once, but this time for only a brief second. Other various whispers continue to surround your mind everywhere else, but they have calmed down slightly.`

Author: I feel this loses a lot of impact if the single word isn't on a line of its own and is just shoved into the rest of the text with everything else, which is why I intentionally separated it. Rest is fine.

> 			`A bizarre image is suddenly flashed into your head, and it feels like millions of eyes are staring at you from beyond the boundaries of space. Turning your head to look around again, you find the source - a strange-looking spacecraft in the vicinity of two of this system's planets. When it spoke, it sounded like thousands of voices speaking at once, but this time for only a brief second. Other various whispers continue to surround your mind everywhere else, but they have calmed down slightly.`

Author: Not sure about viewing the ship, though. I've been told to avoid sensors and radar of the likes, so I wouldn't know what the best way to approach something from that angle would be.

			choice
				`	"Who are you?"`
				`	"Where am I?"`
					goto where

> 			`	The numerous whispers begin to pick up again, but rather than becoming louder, it sounds like they are going back and forth in your head. It almost seems they are having a discussion. After a few more seconds of this, dead silence resumes, and then a single word.`

Reviewer 2: ```suggestion
			`	The numerous whispers begin to pick up again, but rather than becoming louder, it sounds like they are going back and forth in your head. It almost seems they are having a discussion. After a few more seconds of this, dead silence resumes, and then two words.`
```

Author: I feel inclined to roll this one back and only have them say "Precursors", because that, along with "a single word", sounds more mysterious. On second thought, them namedropping themselves doesn't seem very mysterious.

Reviewer 2: I feel like just having them say precursors would be the better option yeah.

> 			`	The numerous whispers begin to pick up again, but rather than becoming louder, it sounds like they are going back and forth in your head. It almost seems they are having a discussion. After a few more seconds of this, dead silence resumes, and then a single word.`

Reviewer 3: ```suggestion
			`	The numerous whispers begin to pick up again, but rather than becoming louder, it sounds like they are going back and forth in your head. It almost seems they are having a discussion. After a few more seconds of this, dead silence resumes and then is broken again by a single word.`
```
"and then is" can be spruced up more as well ("before being" for example, but the construction there with the nebulously implied verb doesn't work well for written text imo.

			`	"P r e c u r s o r s."`
			choice
				`	"Where am I?"`
			label where
			`	Once again, the whispers start to pick up. The swirl of endless vocals continues on in your head for several more seconds, each one more agonizing than the last. Finally, they cease, and you're left in another moment's worth of silence.`
			`	"T h e  E n d."`
			choice
				`	"What do you mean by "The End"?"`

> 			`	The voices resume, but if there's anything you can tell, it's that they've taken a more dreary tone than before. You try once more to communicate with the ship, but despite constant whispering in the background, it no longer answers. "The End" may be referring to some unpleasant event or memory to them.`

Reviewer 6: ```suggestion
			`	The voices resume, but you sense a newfound dreariness in their tone. You try once again to communicate with the ship, but despite constant whispering in the background, it no longer answers.`
```

> 			`Taking another look at the vessel, it seems to be of some sort of advanced origin. Eerily star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating section of parts that hums in a strange tune. The Rulei were probably far more advanced than anything human, even possibly in line with something like the Quarg.`

Reviewer 2: ```suggestion
			`Taking another look at the vessel, it seems to have some sort of advanced origin. Eerily star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating section of parts that hums in a strange tune. The Rulei were probably far more advanced than anything human, even possibly in line with something like the Quarg.`
```

> 			`Taking another look at the vessel, it seems to have some sort of advanced origin. Eerie star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating ring of objects that hum in a strange tune as they rotate. The Rulei were probably far more advanced than anything human, even possibly in line with the Quarg.`

Reviewer 3: ```suggestion
			`Taking another look at the vessel, it seems to have some sort of advanced origin. Eerie star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating ring of objects, which your sensors detect humming in a strange tune as they rotate. The Rulei were probably far more advanced than anything human, even possibly in line with the Quarg.`
```
Still wanting to see clarification on the "hum" from this other ship, so something like this. Or, perhaps instead something along the lines of "and as you watch them rotate, you feel a strange tune humming in the back of your mind."

Author: I'm not sure if it would be a "in your mind" thing since in-game it's a proximity sound that follows the ship around like normal engine sounds, and not an overarching sound like the ambience is. Though that does bring up the question of are pilots able to hear ships outside of their own. I think I'll just omit the sound part and leave the 'strange rotation floaty` bit and reworded it some to fit.

> 			`Taking another look at the vessel, it seems to have some sort of advanced origin. Eerily star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating ring of objects that hum in a strange tune as they rotate. The Rulei were probably far more advanced than anything human, even possibly in line with the Quarg.`

Reviewer 2: ```suggestion
			`Taking another look at the vessel, it seems to have some sort of advanced origin. Eerie star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating ring of objects that hum in a strange tune as they rotate. The Rulei were probably far more advanced than anything human, even possibly in line with the Quarg.`
```

> 			`The planets below seem to have an abnormal appearance similar to that of the starship, but, for some reason, even looking at them gives you a headache. Something intense must have happened here a long time ago.`

Reviewer 2: ```suggestion
			`The planets below seem to have an abnormal appearance similar to that of the starship, but for some reason even looking at them gives you a headache. Something intense must have happened here a long time ago.`
```

				decline
				
				
mission "Rulei Planet"
	invisible
	landing
	source
		attributes "rulei planet"
	on offer
		conversation
			`	It feels as though there is a tremendous weight on your mind while you visit this world. The same sort of purple starlit hues of the alien ship from above can also be found in strange formations on the surface. Unlike the the whispers from before, the voices here are unbearable, especially the closer you get to the surface.`
			`You can barely hear yourself think, and it's enough to drive any living thing mad. However, the most ominous sensation here is that feeling of your consciousness being pulled at from beneath you, like something is attempting to drag your mind down below.`
		
			`It's best you don't remain here for too long.`
				decline

phrase "rulei hostile"
	word

> 		"C o n c e e d."

Reviewer 7: ```suggestion
		"C o n c e d e."
```

# Copyright (c) 2022 by Saugia
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Relic of the Past"
	invisible
	landing
	to complete
		never
	npc save
		government "Rulei"

> 		personality heroic vindictive staying mute unconstrained frugal

Reviewer 4: Why does it have hostile hails now if it's mute?

Author: Kind of forgot that part. I'm wondering if it should remain mute, or have a bunch of lines anyways since they can technically speak your language.

		system "Kanguwa"
		ship "vyu-Ira" "Solemnity"

mission "Ira Contact"
	invisible
	landing
	to complete
		never

> 	on enter Kanguwa

Reviewer 4: ```suggestion
	on enter "Kanguwa"
```

		fail

> 		log "Factions" "Rulei" `The Rulei were likely an ancient, technologically advanced civilization from long ago.`
> 			`They communicate through some sort of invasive psionic ability.`

Reviewer 4: This feels a bit sparse. Let's add some spice.
```suggestion
		log "Factions" "Rulei" `The Rulei were likely an ancient, technologically advanced civilization from long ago, but the only things that remain of them are a single strange ship and two scared planets.`
			`They communicate through some sort of invasive psionic ability, speaking in a thousand voices all at once. Prolonged contact seems unhealthy.`
```

Author: Scared planets? Or is it supposed to be Sacred? Am confusion.

Reviewer 4: Yeah.

		conversation

> 			`Moments after arriving at Kanguwa, you are overwhelmed by an intense sense of emptiness. As you attempt to comprehend this feeling, you begin hearing whispers from behind.`

Reviewer 4: Feels a bit weird to name the system you've only just entered for the first time.
```suggestion
			`Moments after entering the system, you are overwhelmed by an intense sense of emptiness. As you attempt to comprehend this feeling, you begin hearing whispers from behind.`
```

			choice
				`	(Turn around slowly.)`

> 				`	(Pull out your sidearm.)`

Reviewer 4: ```suggestion
				`	(Pull out my sidearm and quickly turn.)`
```

			`	You turn around, but nothing is there besides the familiar internals of your ship. The whispers remain behind your head, like an itch that can't be reached. The whispers are in a completely foreign language that you can't even begin to understand. As you keep looking around, you begin hearing more and more murmuring. There feels to be hundreds of distinct voices tugging your mind in every direction at once now, overloading your senses. Just as you think you can't take anymore, everything falls completely silent.`
			choice
				`	"Hello?"`
				`	"Is there someone there?"`
				`	"Show yourself!"`
			scene "scene/rulei flash"
			`	"Human."`
			`	An image suddenly flashes in your head as you feel millions of eyes staring at you from beyond the boundaries of space. Looking around again, you see a strange-looking spacecraft a bit off into the distance in front of you facing your direction. The voices continue to surround your mind with only slightly reduced intensity.`
			choice
				`	"Who are you?"`
				`	"Where am I?"`
					goto where

> 			`	Numerous voices start up, but rather than becoming louder, they go back and forth in your head, as though they are having a discussion. After a few seconds, they fall silent, then speak a single word.`

Reviewer 4: ```suggestion
			`	Numerous voices start up, but rather than becoming louder, they go back and forth in your head, as though they are having a discussion. After a few seconds, they fall silent, then speak a single word in a cacophonous unison.`
```

			`	"Precursors."`
			choice
				`	"What is this place?"`
				`	"Precursors of what?"`
			label where

> 			`	Once again, the whispers start to pick up with renewed strength. The swirl of endless vocals assaults your head, each moment more agonizing than the last. Finally, they cease, and you are left in silence.`

Reviewer 4: Not sure the "once again" works if you jumped to here from "Where am I?" Dropping it should be fine.
```suggestion
			`	The whispers start to pick up with renewed strength. The swirl of endless vocals assaults your head, each moment more agonizing than the last. Finally, they cease, and you are left in silence.`
```

			`	"An End."`
			choice

> 				`	"An end to what?"'`

Reviewer 4: ```suggestion
				`	"An end to what?"`
```

				`	"What do you mean by that?"`
			`	The voices resume, but you sense a newfound dreariness in their tone. You try once again to communicate with the ship, but despite constant whispering in the background, it no longer answers.`

> 			`Taking another look at the vessel, it seems to have some sort of advanced origin. Eerie star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating ring of objects that rotate in a pecurliar manner. The Rulei were probably far more advanced than anything human, even possibly in line with the Quarg.`
> 			`The planets in this system seem to have an abnormal appearance similar to that of the starship, but for some reason even looking at them gives you a headache. Something unusual must have happened here a long time ago.`

Reviewer 4: Indentation. Also, I wouldn't mention the Quarg, Leave the technology level of the Rulei up to the player's imagination.
```suggestion
			`	Taking another look at the vessel, it seems to have some sort of advanced origin. Eerie star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating ring of objects that rotate in a peculiar manner.`
			`	The planets in this system seem to have an abnormal appearance similar to that of the starship, but for some reason even looking at them gives you a headache. Something unusual must have happened here a long time ago.`
```

				decline
				
				
mission "Rulei Planet"
	invisible
	landing
	source

> 		attributes "rulei planet"

Reviewer 4: ```suggestion
		attributes "rulei"
```

	on offer
		conversation

> 			`	It feels as though there is a tremendous weight on your mind while you visit this world. The same sort of purple starlit hues of the alien ship from above can also be found in various strange formations across the land. Unlike the the whispers from before, the voices here are painfully unbearable, especially the closer you get to the surface.`
> 			`You can barely hear yourself think, and it's enough to drive any living thing mad. However, the most ominous sensation here is a persistent feeling of your consciousness being pulled at from beneath you, like something is attempting to drag your mind into the planet below.`
> 		
> 			`It's best you don't remain here for too long.`

Reviewer 4: What's with the inverted spacing? I imagine it was a mistake, but it has me thinking that we could stylize this strangely to fit the mood.
```suggestion
			`It feels as though there is a tremendous weight on your mind while you visit this world. The same sort of purple starlit hues of the alien ship from above can also be found in various strange formations across the land. Unlike the whispers from before, the voices here are painfully unbearable, especially the closer you get to the surface.`
			`	You can barely hear yourself think.`
			`											It's enough to drive any living thing mad.`
			`	The most ominous sensation here is a persistent feeling of your consciousness being pulled at from beneath you.
			`	Like something is attempting to drag`
			`																					your`
			`																							mind into the planet below.`
			``
			``
			``
			`	It's best you don't remain here for too long.`
```
![image](https://user-images.githubusercontent.com/17688683/215307210-a3dac4fd-9b13-4abd-aaed-d8e560d256b3.png)

Author: I like it. Very spooky. I just hope it doesn't downplay Archon telepathy somehow. Or maybe it should, since this is far from one singular entity.

Author: Tests are being funky with an end quote tho.

# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Relic of the Past"
	invisible
	landing
	to complete
		never
	npc save
		government "Rulei"
		personality heroic vindictive staying mute unconstrained frugal
		system "Kanguwa"
		ship "vyu-Ira" "Solemnity"

mission "Ira Contact"
	invisible
	landing
	to complete
		never
	on enter "Kanguwa"
		fail
		log "Factions" "Rulei" `The Rulei were likely an ancient, technologically advanced civilization from long ago, but the only things that remain of them are a single strange ship and two scarred planets.`
			`They communicate through some sort of invasive psionic ability, speaking in a thousand voices all at once. Prolonged contact seems unhealthy.`
		conversation
			`Moments after entering the system, you are overwhelmed by an intense sense of emptiness. As you attempt to comprehend this feeling, you begin hearing whispers from behind.`
			choice
				`	(Turn around slowly.)`

> 				`	(Pull out my sidearm and quickly turn.)`

Reviewer 2: ```suggestion
				`	(Pull out my sidearm and turn quickly.)`
```

			`	Taking another look at the vessel, it seems to have some sort of advanced origin. Eerie star-like patterns flow on its surface, and it doesn't visibly have any standard means of propulsion. At the rear end is some type of floating ring of objects that rotate in a peculiar manner.`
			`	The planets in this system seem to have an abnormal appearance similar to that of the starship, but for some reason even looking at them gives you a headache. Something unusual must have happened here a long time ago.`
				decline
				
				
mission "Rulei Planet"
	invisible
	landing
	source
		attributes "rulei"
	on offer
		conversation
			`It feels as though there is a tremendous weight on your mind while you visit this world. The same sort of purple starlit hues of the alien ship from above can also be found in various strange formations across the land. Unlike the whispers from before, the voices here are painfully unbearable, especially the closer you get to the surface.`
			`	You can barely hear yourself think.`
			`											It's enough to drive any living thing mad.`

> 			`	The most ominous sensation here is a persistent feeling of your consciousness being pulled at from beneath you.

Reviewer 4: ```suggestion
			`	The most ominous sensation here is a persistent feeling of your consciousness being pulled at from beneath you.`
```
